onadata/libs/utils/common_tools.py

```diff
@@ -104,3 +104,28 @@ def get_response_content(response):
         contents = response.content
 
     return contents
+
+
+def json_stream(data, json_string):
+    """
+    Generator function to stream JSON data
+    """
+
```

Comment:
since we're always going to be dumping the JSON, let's move that out of the passed in function and into here, a way we could do that without repeating the dump call, assuming we rename the argument to `item_transform_fn`, is:

```python
json_dumps_fn = lambda x: json.dumps(item_transform_fn(x))
```

and we replace `json_string` below with `json_dumps_fn`